zebra-rpc/src/methods/get_block_template_rpcs.rs

```diff
@@ -58,10 +66,26 @@ pub trait GetBlockTemplateRpc {
     /// - This rpc method is available only if zebra is built with `--features getblocktemplate-rpcs`.
     #[rpc(name = "getblocktemplate")]
     fn get_block_template(&self) -> BoxFuture<Result<GetBlockTemplate>>;
+
+    /// Submits block to the node to be validated and committed.
+    /// Returns the [`submit_block::Response`] for the transaction, as a JSON string.
```

Comment:
```suggestion
    /// Returns the [`submit_block::Response`] for the operation, as a JSON string.
```
I think it can be a bit confusing to say transaction here, maybe operation?